cmd/kube-apiserver/app/server.go

```diff
@@ -240,6 +241,8 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.KubeletConfig.CertFile, "kubelet-client-certificate", s.KubeletConfig.CertFile, "Path to a client cert file for TLS.")
 	fs.StringVar(&s.KubeletConfig.KeyFile, "kubelet-client-key", s.KubeletConfig.KeyFile, "Path to a client key file for TLS.")
 	fs.StringVar(&s.KubeletConfig.CAFile, "kubelet-certificate-authority", s.KubeletConfig.CAFile, "Path to a cert. file for the certificate authority.")
+	// TODO: delete this flag when we stop fixing update requests at server-side. See #14126
+	fs.BoolVar(&validation.FixUpdateAtServer, "fix-update", true, "If true, server will do its best to fix the update request to pass the validation, e.g., setting empty UID in update request to its existing value.")
```

Comment:
how about "repair-malformed-updates" as a name? and in the text, say we expect to delete the flag as soon as we identify and fix all such clients.

Default to false--let's see if the PR builder is happy. If is, maybe we don't need to flag protect at all.